### ISSUE TITLE
Mask Scoring R-CNN loss

### ISSUE THREAD

**Issue by BloomBabe** [open-mmlab/mmdetection#2465]
Do I understand correctly that the calculation of the cross-entropy loss function in the mask head  is based on  the only classification score, and not according to the score_mask = score_cls*score_iou?
